### PR TITLE
Feat: consider user as guest and login automatically

### PR DESCRIPTION
**Pull request by rajatkantinandi**
- [Expand question card after save](https://github.com/rajatkantinandi/quizmaster/commit/814e5d97f1172293aa460997efdb37c9c92924ed)

### CODE REVIEW

File: src/routes/ConfigureQuiz/QuestionCard.tsx [hunk 1]
```diff
@@ -92,7 +93,7 @@ export default function QuestionCard({
     updateQuestionData(idx, question);
     await updateQuizData();
     setActiveQuestionIndex(null);
-    setExpandedQuestionIndex(null);
```
Comment (govind007) [inline src/routes/ConfigureQuiz/QuestionCard.tsx:95]: why this change ?

**Review by govind007 — APPROVED** [review]

File: src/routes/ConfigureQuiz/QuestionCard.tsx (hunk 1 above)
Comment (rajatkantinandi) [reply src/routes/ConfigureQuiz/QuestionCard.tsx:95]: Want to keep the question expanded after edit so that they can preview the saved question as we don't have preview mode anymore.

Comment (rajatkantinandi) [reply src/routes/ConfigureQuiz/QuestionCard.tsx:95]: When they one more question or expand another question then that one will be collapsed automatically.
